catalogs/catalog-lakehouse-paimon/src/test/java/com/datastrato/gravitino/catalog/lakehouse/paimon/TestPaimonCatalog.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.lakehouse.paimon;
+
+import static com.datastrato.gravitino.catalog.lakehouse.paimon.PaimonCatalog.CATALOG_PROPERTIES_META;
+import static com.datastrato.gravitino.catalog.lakehouse.paimon.PaimonCatalog.SCHEMA_PROPERTIES_META;
+
+import com.datastrato.gravitino.Namespace;
+import com.datastrato.gravitino.catalog.PropertiesMetadataHelpers;
+import com.datastrato.gravitino.catalog.lakehouse.paimon.ops.PaimonCatalogOps;
+import com.datastrato.gravitino.connector.CatalogOperations;
+import com.datastrato.gravitino.connector.HasPropertyMetadata;
+import com.datastrato.gravitino.connector.PropertiesMetadata;
+import com.datastrato.gravitino.meta.AuditInfo;
+import com.datastrato.gravitino.meta.CatalogEntity;
+import com.google.common.collect.Maps;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestPaimonCatalog {
+
+  static final HasPropertyMetadata PAIMON_PROPERTIES_METADATA =
+      new HasPropertyMetadata() {
+
+        @Override
+        public PropertiesMetadata tablePropertiesMetadata() throws UnsupportedOperationException {
+          throw new UnsupportedOperationException("Table properties are not supported");
+        }
+
+        @Override
+        public PropertiesMetadata catalogPropertiesMetadata() throws UnsupportedOperationException {
+          return CATALOG_PROPERTIES_META;
+        }
+
+        @Override
+        public PropertiesMetadata schemaPropertiesMetadata() throws UnsupportedOperationException {
+          return SCHEMA_PROPERTIES_META;
+        }
+
+        @Override
+        public PropertiesMetadata filesetPropertiesMetadata() throws UnsupportedOperationException {
+          throw new UnsupportedOperationException("Fileset properties are not supported");
+        }
+
+        @Override
+        public PropertiesMetadata topicPropertiesMetadata() throws UnsupportedOperationException {
+          throw new UnsupportedOperationException("Topic properties are not supported");
+        }
+      };
+
+  @Test
+  public void testListDatabases() {
```

Comment:
why putting `testListDatabases` here?

Reply:
just to make sure the catalog object works.

Reply:
maybe you should use a better name

Reply:
updated.